src/sage/rings/polynomial/polydict.pyx

```diff
@@ -128,6 +116,9 @@ cdef class PolyDict:
 
         - ``force_etuples`` -- deprecated
 
+        - ``check`` -- if set to ``False`` then assumes that the exponents are all valid ``ETuple``. In
+          that case the construction is a bit faster.
```

Comment:
```suggestion
        - ``check`` -- if set to ``False`` then assumes that the exponents are all valid ``ETuple``; in
          that case the construction is a bit faster
```

Reply:
Fixed in https://github.com/sagemath/sage/pull/35174/commits/1a7eeac1a5da609b30fa9f0a36d6784e5dff6e7d